src/components/common/FileUploadPdf.vue

```diff
@@ -205,15 +205,15 @@ export default class FileUploadPdf extends Vue {
     try {
       const arrayBuffer = await file.arrayBuffer()
       const data = new Uint8Array(arrayBuffer) // put it in a Uint8Array
-      const document = await this.pdfjsLib.getDocument({ data: data })
+      const document = await this.pdfjsLib.getDocument({ data }).promise
       const perms = await document.getPermissions()
       return { isEncrypted: false, isContentLocked: !!perms }
-    } catch (err) {
-      if (err.name === 'PasswordException') {
+    } catch (error) {
+      if (error.name === 'PasswordException') {
         return { isEncrypted: true, isContentLocked: true }
       }
-      throw err // re-throw any other error
```

Comment:
If the error isn't "PasswordException", don't we want to re-throw it?

Reply:
Let's go with your PR Sev in which you only added the `.promise`. That should be good. I think that resolves your 2 comments.

Reply:
OK.